test/unittest/StatisticsBackend/StatisticsBackendTests.cpp

```diff
@@ -96,6 +96,8 @@ TEST_F(statistics_backend_tests, get_info)
         info.erase(ALIAS_INFO_TAG);
         EXPECT_EQ(entity->active, info[ALIVE_INFO_TAG]);
```

Comment:
This is tested, but actually wrongly tested, as it is not stored in the dump

Reply:
:+1: